ivy.el

```diff
@@ -3954,10 +3954,10 @@ When `ivy-calling' isn't nil, call `ivy-occur-press'."
     (ivy-occur-press)))
 
 (define-derived-mode ivy-occur-mode fundamental-mode "Ivy-Occur"
-  "Major mode for output from \\[ivy-occur].
+                     "Major mode for output from \\[ivy-occur].
```

Comment:
`define-derived-mode` isn't even Common Lisp; the string being indented is a docstring, not a formal argument.